### PR TITLE
修改API映射关系文档的bug

### PR DESCRIPTION
**Pull request by ROckDog22**
### PR types
Others
### PR changes
Others
### Descriptions
fix doc bugs about Pacovert.

### CODE REVIEW

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.unique.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-## torch.Tensor.unique
+## [ 参数不一致 ]torch.Tensor.unique
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.unique.md]: 这个应该是torch参数更多

File: docs/guides/model_convert/convert_from_pytorch/api_difference/functional/torch.nn.functional.cross_entropy.md [hunk 2]
```diff
@@ -43,16 +43,10 @@ paddle.nn.functional.cross_entropy(input,
 ### 转写示例
 size_average
 ```python
-# Pytorch 的 size_average 、reduce 参数转为 Paddle 的 reduction 参数
-if size_average is None:
-    size_average = True
-if reduce is None:
-    reduce = True
+# Pytorch 写法
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/functional/torch.nn.functional.cross_entropy.md:47]: 多写几种映射case，因为也有不同的size_average 、reduce不同的取值

File: docs/guides/model_convert/convert_from_pytorch/api_difference/functional/torch.nn.functional.cross_entropy.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
-## [ torch 参数更多 ]torch.nn.functional.cross_entropy
+## [ 参数不一致 ]torch.nn.functional.cross_entropy
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/functional/torch.nn.functional.cross_entropy.md]: 这个是torch参数更多

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.CrossEntropyLoss.md [hunk 4]
```diff
@@ -1,4 +1,4 @@
-## [torch 参数更多]torch.nn.CrossEntropyLoss
+## [参数不一致]torch.nn.CrossEntropyLoss
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.CrossEntropyLoss.md]: 这个是torch参数更多

下面的例子可以多写几种映射case

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.LSTM.md [hunk 5]
```diff
@@ -37,7 +37,6 @@ paddle.nn.LSTM(input_size,
 | bias          | -            | 是否使用偏置，Paddle bias_ih_attr 和 bias_hh_attr 可实现相应功能。  |
 | batch_first   | time_major   | PyTorch 表示 batch size 是否为第一维，PaddlePaddle 表示 time steps 是否为第一维，它们的意义相反。  |
 | dropout   | dropout   | 表示 dropout 概率。  |
-| batch_first   | time_major   | PyTorch 表示 batch size 是否为第一维，PaddlePaddle 表示 time steps 是否为第一位，它们的意义相反。  |
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.LSTM.md:40]: torch有这个参数

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.MSELoss.md [hunk 6]
```diff
@@ -24,16 +24,9 @@ paddle.nn.MSELoss(reduction='mean')
 ### 转写示例
 #### size_average
 ```python
-# Pytorch 的 size_average、reduce 参数转为 Paddle 的 reduction 参数
-if size_average is None:
-    size_average = True
-if reduce is None:
-    reduce = True
+# Paddle 写法
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.MSELoss.md:27]: 可以多写几种映射case

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.NLLLoss.md [hunk 7]
```diff
@@ -33,16 +33,9 @@ Pytorch 相比 Paddle 支持更多其他参数，具体如下：
...
-    reduce = True
-
-if size_average and reduce:
-    reduction = 'mean'
-elif reduce:
-    reduction = 'sum'
-else:
-    reduction = 'none'
+# Paddle 写法
+torch.nn.NLLLoss(size_average=True)
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.NLLLoss.md:37]: 多写几种映射case，因为也有不同的size_average 、reduce不同的取值

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.SmoothL1Loss.md [hunk 8]
```diff
@@ -28,16 +28,9 @@ paddle.nn.SmoothL1Loss(reduction='mean',
 ### 转写示例
 #### size_average
 ```python
-# Pytorch 的 size_average、reduce 参数转为 Paddle 的 reduction 参数
-if size_average is None:
-    size_average = True
-if reduce is None:
-    reduce = True
+# Paddle 写法
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.SmoothL1Loss.md:31]: 多写几种映射case，因为也有不同的size_average 、reduce不同的取值

File: docs/guides/model_convert/convert_from_pytorch/pytorch_api_mapping_cn.md [hunk 9]
```diff
@@ -100,8 +100,8 @@
 | 69   |  [torch.is_complex](https://pytorch.org/docs/stable/generated/torch.is_complex.html?highlight=is_complex#torch.is_complex)  |  [paddle.is_complex](https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/is_complex_cn.html#is-complex)  |    功能一致, 仅参数名不一致 , [差异对比](https://github.com/PaddlePaddle/docs/tree/develop/docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.is_complex.md)  |
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/pytorch_api_mapping_cn.md:100]: 这个表和外面的保持一致即可

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.unique.md (hunk 1 above)
Comment (ROckDog22) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.unique.md]: 已修改

File: docs/guides/model_convert/convert_from_pytorch/api_difference/functional/torch.nn.functional.cross_entropy.md (hunk 2 above)
Comment (ROckDog22) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/functional/torch.nn.functional.cross_entropy.md:47]: 已修改

File: docs/guides/model_convert/convert_from_pytorch/api_difference/functional/torch.nn.functional.cross_entropy.md (hunk 3 above)
Comment (ROckDog22) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/functional/torch.nn.functional.cross_entropy.md]: 已修改

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.CrossEntropyLoss.md (hunk 4 above)
Comment (ROckDog22) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.CrossEntropyLoss.md]: 已修改

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.LSTM.md (hunk 5 above)
Comment (ROckDog22) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.LSTM.md:40]: 这里重复了
image
所以删除了

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.MSELoss.md (hunk 6 above)
Comment (ROckDog22) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.MSELoss.md:27]: 已修改

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.NLLLoss.md (hunk 7 above)
Comment (ROckDog22) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.NLLLoss.md:37]: 已修改

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.SmoothL1Loss.md (hunk 8 above)
Comment (ROckDog22) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.SmoothL1Loss.md:31]: 已修改

File: docs/guides/model_convert/convert_from_pytorch/pytorch_api_mapping_cn.md (hunk 9 above)
Comment (ROckDog22) [reply docs/guides/model_convert/convert_from_pytorch/pytorch_api_mapping_cn.md:100]: 这里面确实有好多错误:joy:
image
这个是写了两种映射，保留了正确的，对应的api也要改

Comment (ROckDog22) [reply docs/guides/model_convert/convert_from_pytorch/pytorch_api_mapping_cn.md:100]: image
这种写的重复了

Comment (ROckDog22) [reply docs/guides/model_convert/convert_from_pytorch/pytorch_api_mapping_cn.md:100]: 因为修改了一部分API，对应的映射文档也修改了

Comment (ROckDog22) [reply docs/guides/model_convert/convert_from_pytorch/pytorch_api_mapping_cn.md:100]: image
这种属于url写错了

**Comment by zhwesky2010** [comment]
冲突了需要处理下

**Comment by ROckDog22** [comment]
> 冲突了需要处理下

已修改

File: docs/guides/model_convert/convert_from_pytorch/pytorch_api_mapping_cn.md [hunk 10]
```diff
@@ -189,8 +189,8 @@
 | 158   | [torch.diag\_embed](https://pytorch.org/docs/stable/generated/torch.diag_embed.html?highlight=diag_embed#torch.diag_embed) | [paddle.nn.functional.diag\_embed](https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/nn/functional/diag_embed_cn.html#diag-embed) | 功能一致，参数完全一致    |
 | 159    | [torch.is\_grad\_enabled](https://pytorch.org/docs/stable/generated/torch.is_grad_enabled.html?highlight=is_grad_enabled#torch.is_grad_enabled) | [paddle.is\_grad\_enabled](https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/is_grad_enabled_cn.html#is-grad-enabled) | 功能一致，无参数   |
 | 160   | [torch.nansum](https://pytorch.org/docs/stable/generated/torch.nansum.html?highlight=nansum#torch.nansum) | paddle.nansum | 功能一致，仅参数命名不一致    |
-| 161   | [torch.meshgrid](https://pytorch.org/docs/stable/generated/torch.meshgrid.html?highlight=meshgrid#torch.meshgrid) | [paddle.meshgrid](https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/meshgrid_cn.html#meshgrid) | 功能一致，torch 参数更多 [差异对比](https://github.com/PaddlePaddle/docs/blob/develop/docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.meshgrid.md) |
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/pytorch_api_mapping_cn.md:192]: 这两个被删掉了吗，后面还是要补回来

File: docs/guides/model_convert/convert_from_pytorch/pytorch_api_mapping_cn.md [hunk 11]
```diff
@@ -189,8 +189,8 @@
...
-| 162   | [torch.sigmoid](https://pytorch.org/docs/stable/generated/torch.sigmoid.html?highlight=sigmoid#torch.sigmoid) | [paddle.nn.functional.sigmoid](https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/nn/functional/sigmoid_cn.html#sigmoid) | 功能一致，torch 参数更多 [差异对比](https://github.com/PaddlePaddle/docs/blob/develop/docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.sigmoid.md) |
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/pytorch_api_mapping_cn.md:193]: 这两个被删掉了吗，后面还是要补回来

**Review by zhwesky2010 — APPROVED** [review]

File: docs/guides/model_convert/convert_from_pytorch/pytorch_api_mapping_cn.md (hunk 10 above)
Comment (ROckDog22) [reply docs/guides/model_convert/convert_from_pytorch/pytorch_api_mapping_cn.md:192]: image
image
这个是原始文档的doc写重复了，这两个API写了两遍，删除做了修改

Comment (ROckDog22) [reply docs/guides/model_convert/convert_from_pytorch/pytorch_api_mapping_cn.md:192]: 不需要补回来，
